main/acle_pdf_conf.yaml

```diff
@@ -0,0 +1,16 @@
+title: Arm C Language Extensions
+subtitle: 2021Q2
+date: 02 July 2021
+logo: ./tools/Arm-logo-blue-RGB.pdf
+documentclass: article
+toc: true
+numbersections: true
+geometry: landscape,top=3cm,bottom=3cm,left=2cm,right=2cm
+fontfamily: helvet
+classoption: 10pt
+fontcolor: darkgray
+colorlinks: citecolor=darkgray
+copyright-text: Copyright © 2020-2021, Arm Limited and its affiliates. All rights reserved.
+chapter-formatting: true
+papersize: a3
```

Comment:
why a3? it should be a4

Reply:
The table cell text was overflowing into the adjacent cell in a4 format

Reply:
Which of the tables? It is just one or more? Can we re-arrange the table to fix the issue?

Reply:
<img width="581" alt="Screenshot 2021-11-05 164451" src="https://user-images.githubusercontent.com/90907783/140547294-be8ce854-3941-4d3d-b068-4a3cadd3d6c4.png">

Reply:
There's a decent few rows like this